lib/ssh_scan/api.rb

```diff
@@ -0,0 +1,48 @@
+require 'sinatra/base'
+require 'sinatra/namespace'
+require 'ssh_scan/version'
+require 'ssh_scan/policy'
+require 'ssh_scan/scan_engine'
+require 'json'
+
+module SSHScan
+  class API < Sinatra::Base
```

Comment:
Adding this line after your class definition will give you the ability to set the port...

    set :port, 80

You'll probably want to make it user configurable and set to something other than 4567 by default, just to give it a more polished feel.